schemas/org/list-orgs-response.json

```diff
@@ -79,12 +79,12 @@
               "created": {
                 "type": "string",
                 "format": "date-time",
-                "description": "The time the organization was created."
+                "description": "The time the organization was created'
               },
               "modified": {
                 "type": "string",
                 "format": "date-time",
-                "description": "The last time the organization was modified."
+                "description": "The last time the organization was modified'
```

Comment:
I think the descriptions end with a single quote now but should end with a double quote.